Makefile

```diff
@@ -12,42 +12,67 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The controller-gen command for generating CRDs from API definitions.
-CONTROLLER_GEN=GOFLAGS=-mod=vendor go run sigs.k8s.io/controller-tools/cmd/controller-gen
-# The output directory for generated CRDs.
-CRD_OUTPUT_DIR ?= "config/crd/bases"
-# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:crdVersions=v1"
+DOCKER ?= docker
+# Image to build protobugs
+PROTO_IMG ?= k8s.gcr.io/kube-cross:v1.13.6-1
+# TOP is the current directory where this Makefile lives.
+TOP := $(dir $(firstword $(MAKEFILE_LIST)))
+# ROOT is the root of the mkdocs tree.
+ROOT := $(abspath $(TOP))
 
-all: controller docs
+all: controller generate verify
 
-# Kubebuilder driven custom resource definitions.
+# Build manager binary and run static analysis.
 .PHONY: controller
 controller:
-	make -f kubebuilder.mk
+	make -f kubebuilder.mk manager
```

Comment:
Reference `make` as `$(MAKE)`. 

Re: http://www.chiark.greenend.org.uk/doc/make-doc/make.html/Makefile-Conventions.html

```
The Makefile rules for building and installation can also use compilers and related programs, but should do so via make variables so that the user can substitute alternatives. Here are some of the programs we mean:

ar bison cc flex install ld ldconfig lex
make makeinfo ranlib texi2dvi yacc
Use the following make variables to run those programs:

$(AR) $(BISON) $(CC) $(FLEX) $(INSTALL) $(LD) $(LDCONFIG) $(LEX)
$(MAKE) $(MAKEINFO) $(RANLIB) $(TEXI2DVI) $(YACC)
```

Reply:
@frobware thanks for the review. I pushed commit `f43ee37` to address your feedback.